judgments/templatetags/query_filters.py

```diff
@@ -16,11 +16,46 @@ def make_query_string(params):
     return "&".join(pairs)
 
 
+@register.filter
+def removable_filter_param(key):
```

Comment:
can you give some more context around what a `removable_filter_param` is?

Reply:
aha, so theses are the parameters in the query which are 'removable' using the little yellow 'filter control boxes' on the search page. some elements of the query (court, dates, query term) should be able to be removed to widen the search, others (per_page, order, feature flag state) should be ignored. This is a test to check that a parameter should indeed be displayed as a 'remove' button.

Reply:
I see!